### PR TITLE
First branch

### PR DESCRIPTION
**Pull request by TarasTsyhanko**

### CODE REVIEW

File: src/main/java/elements/PageElement.java [hunk 1]
```diff
@@ -1,70 +1,51 @@
...
+
+    Point getLocation();
+
+    Dimension getSize();
+
+    Rectangle getRect();
+
+    String getCssValue(String var1);
+
+    PageElement waitForVisibility();
```
Comment (MykhailoTsiganko) [inline src/main/java/elements/PageElement.java]: rename to waitUntilVisible

File: src/main/java/elements/PageElement.java [hunk 2]
```diff
@@ -1,70 +1,51 @@
...
+
+    PageElement waitForClickable();
```
Comment (MykhailoTsiganko) [inline src/main/java/elements/PageElement.java]: rename to waitUntilClickable

File: src/main/java/elements/PageElement.java [hunk 3]
```diff
@@ -1,70 +1,51 @@
...
+
+    PageElement waitForPresenceOfElementLocated();
```
Comment (MykhailoTsiganko) [inline src/main/java/elements/PageElement.java]: rename to waitUntilPresent

File: src/main/java/elements/PageElementImpl.java [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+public class PageElementImpl implements PageElement {
+    protected final int DEFAULT_VISIBILITY_TIME = 20;
+    protected WebElement element;
```
Comment (MykhailoTsiganko) [inline src/main/java/elements/PageElementImpl.java:13]: replace this element with method getWebElement()

Comment (TarasTsyhanko) [reply src/main/java/elements/PageElementImpl.java:13]: you mean create getter for WebElement?

File: src/main/java/elements/PageElement.java (hunk 2 above)
Comment (TarasTsyhanko) [reply src/main/java/elements/PageElement.java]: fixed

File: src/main/java/elements/PageElement.java (hunk 1 above)
Comment (TarasTsyhanko) [reply src/main/java/elements/PageElement.java]: fixed

File: src/main/java/elements/PageElement.java (hunk 3 above)
Comment (TarasTsyhanko) [reply src/main/java/elements/PageElement.java]: fixed

**Pull request merged**